git_machete/cli.py

```diff
@@ -45,304 +44,517 @@
+
+    subparsers.add_parser(
+        'file',
+        usage=argparse.SUPPRESS,
+        add_help=False,
+        parents=[common_args_parser])
+
+    fork_point_parser = subparsers.add_parser(
+        'fork-point',
+        argument_default=argparse.SUPPRESS,
+        usage=argparse.SUPPRESS,
+        add_help=False,
+        parents=[common_args_parser])
+    fork_point_parser.add_argument('branch', nargs='?')
+    fork_point_exclusive_optional_args = fork_point_parser.add_mutually_exclusive_group()
+    fork_point_exclusive_optional_args.add_argument('--inferred', action='store_true')
+    fork_point_exclusive_optional_args.add_argument('--override-to')
+    fork_point_exclusive_optional_args.add_argument('--override-to-inferred', action='store_true')
+    fork_point_exclusive_optional_args.add_argument('--override-to-parent', action='store_true')
+    fork_point_exclusive_optional_args.add_argument('--unset-override', action='store_true')
+
+    subparsers.add_parser(
+        'format', add_help=False, usage=argparse.SUPPRESS, parents=[common_args_parser])
+
+    github_parser = subparsers.add_parser(
+        'github',
+        argument_default=argparse.SUPPRESS,
+        usage=argparse.SUPPRESS,
+        add_help=False,
+        parents=[common_args_parser])
+    github_parser.add_argument(
+        'subcommand', choices=['anno-prs', 'checkout-prs', 'create-pr', 'retarget-pr'])
+    github_parser.add_argument('pr_no', nargs='?')
+    github_parser.add_argument('--draft', action='store_true')
+
+    go_parser = subparsers.add_parser(
+        'go',
+        aliases=['g'],
+        usage=argparse.SUPPRESS,
+        add_help=False,
+        parents=[common_args_parser])
+    go_parser.add_argument(
+        'direction',
+        choices=['d', 'down', 'f', 'first', 'l', 'last', 'n', 'next',
+                 'p', 'prev', 'r', 'root', 'u', 'up']
+    )
+
+    help_parser = subparsers.add_parser(
+        'help', add_help=False, usage=argparse.SUPPRESS, parents=[common_args_parser])
+    help_parser.add_argument('topic_or_cmd', nargs='?', choices=list(short_docs.keys()))
+
+    subparsers.add_parser(
+        'hooks',
+        usage=argparse.SUPPRESS,
+        add_help=False,
+        parents=[common_args_parser])
+
+    is_managed_parser = subparsers.add_parser(
+        'is-managed',
+        usage=argparse.SUPPRESS,
+        add_help=False,
+        parents=[common_args_parser])
+    is_managed_parser.add_argument('branch', nargs='?')
+
+    list_parser = subparsers.add_parser(
+        'list', usage=argparse.SUPPRESS, add_help=False, parents=[common_args_parser])
+    list_parser.add_argument(
+        'category',
+        choices=[
+            'addable', 'managed', 'slidable', 'slidable-after', 'unmanaged',
+            'with-overridden-fork-point']
+    )
+    list_parser.add_argument('branch', nargs='?', default=argparse.SUPPRESS)
+
+    log_parser = subparsers.add_parser(
+        'log',
+        aliases=['l'],
```

Comment:
Whoops `git machete l` just does nothing and exits 😯 same for `git machete e`, `git machete g <something>`, `git machete s`